packages/design-system-react/src/components/text/Text.types.ts

```diff
@@ -0,0 +1,102 @@
+export type TextProps = {
+  variant?: TextVariant;
+  children: React.ReactNode;
+  className?: string;
+  fontWeight?: FontWeight;
+  fontStyle?: FontStyle;
+  textTransform?: TextTransform;
+  textAlign?: TextAlign;
+  overflowWrap?: OverflowWrap;
+  ellipsis?: boolean;
+  as?: ValidTag;
+  color?: TextColor;
```

Comment:
Can you add descriptions and default values?

Reply:
Great catch! Updated https://github.com/MetaMask/metamask-design-system/pull/110/commits/1d647ecd4eabc33aeafa1a5377ac1f1122966a56

Reply:
To align with best practices, can you add `Optional` to the description of optional props?

Reply:
Added here https://github.com/MetaMask/metamask-design-system/pull/110/commits/4d6fb4864bce9b2cbd11aae129ee6b3457c6ae1c